docs/source/overview.rst

```diff
@@ -2,37 +2,29 @@
 Overview
 ========
 
-What does it currently do
+Neo-python is an implementation of the Neo protocol using Python. 
+
+
+What neo-python does
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- This project aims to be a full port of the original C# `NEO project <https://github.com/neo-project>`_
-- Run a Python based P2P node
-- Interactive CLI for configuring node and inspecting blockchain
-- Compile, test, deploy and run Smart Contracts written in python or any Smart Contract in the ``.avm`` format
-- `NEP2 <https://github.com/neo-project/proposals/blob/master/nep-2.mediawiki>`_ and `NEP5 <https://github.com/neo-project/proposals/blob/master/nep-5.mediawiki>`_ compliant wallet functionality
```

Comment:
Can you either return this line or rephrase it? I'm fine with simplifying text for new users, but at the same time I want to make sure we also address more advanced users that know what they're looking for. This is one of those lines which I think does exactly that.